hive/runners/base.py

```diff
@@ -41,8 +43,13 @@ def __init__(
                 If this is -1, testing is not run.
             test_episodes (int): How many episodes to run testing for.
         """
-        self._environment = environment
-        self._environment.seed(seeder.get_new_seed("environment"))
+        self._train_environment = environment
+        self._train_environment.seed(seeder.get_new_seed("environment"))
+        self._eval_environment = eval_environment
+        if self._eval_environment is not None:
+            self._eval_environment.seed(seeder.get_new_seed("environment"))
+        self._environment = self._train_environment
```

Comment:
Why do we have both `self._environment` and `self._train_environment`?

Reply:
Good catch. I should remove self._environment